### DISCUSSION TITLE
Is it possible to ignore certain keywords, such as import?

### DISCUSSION THREAD

**Discussion by aa563011992** [jplag/JPlag#2089] [Q&A]
Is it possible to ignore certain keywords, such as import?
Now imports are counted as duplicates, even if the import paths are different

**Comment by tsaglam** [comment]
No, this is not possible. JPlag compares the structure of programs, thus abstracting from certain details. Consequently, the code is matched if the structure is identical, even if, for example, the import does not match. This however, is intended, as this allows to detect structural similarities between programs, even if superficial details such as import paths are different. We also do not consider, names, types, and values, for example. The goal of JPlag is to identify potential plagiarism or code similarity based on the underlying logic and structure, not on surface-level differences.
